apps/counter2/app.js

```diff
@@ -9,23 +9,23 @@ var s = Object.assign({
   colortext: true,
 }, require('Storage').readJSON("counter2.json", true) || {});
 
-f1 = (s.colortext) ? "#f00" : "#fff";
-f2 = (s.colortext) ? "#00f" : "#fff";
-b1 = (s.colortext) ? g.theme.bg : "#f00";
-b2 = (s.colortext) ? g.theme.bg : "#00f";
+const f1 = (s.colortext) ? "#f00" : "#fff";
+const f2 = (s.colortext) ? "#00f" : "#fff";
+const b1 = (s.colortext) ? g.theme.bg : "#f00";
+const b2 = (s.colortext) ? g.theme.bg : "#00f";
 
 var counter = 0;
 var drag;
 
-screenwidth = g.getWidth();
-screenheight = g.getHeight();
-halfwidth = screenwidth / 2;
-halfheight = screenheight / 2;
+const screenwidth = g.getWidth();
+const screenheight = g.getHeight();
+const halfwidth = screenwidth / 2;
+const halfheight = screenheight / 2;
 
-counter = [];
+counter = []; // FIXME: How does this relate to `var counter = 0;` 8 lines above this one.
```

Comment:
I suppose the `= 0` is a dead store and we can just delete the above line and change this like so:
```suggestion
var counter = [];
```